plaso/parsers/mactime.py

```diff
@@ -26,7 +26,7 @@ class MactimeEventData(events.EventData):
     filename (str): name of the file.
     inode (int): "inode" of the file. Note that inode is an overloaded term
         in the context of mactime and used for MFT entry index values as well.
-    md5_hash (str): MD5 hash of the file content.
+    md5 (str): MD5 hash of the file content.
```

Comment:
maybe change to:
MD5 hash of the file content, formatted as a hexadecimal string.